metadata-ingestion/src/datahub/ingestion/source/salesforce.py

```diff
@@ -287,7 +287,13 @@ def __init__(self, config: SalesforceConfig, ctx: PipelineContext) -> None:
 
         except Exception as e:
             logger.error(e)
-            raise ConfigurationError("Salesforce login failed") from e
+            if "API_CURRENTLY_DISABLED" in str(e):
```

Comment:
This string will be present **only if** the `Exception` is actually:
`simple_salesforce.exceptions.SalesforceAuthenticationFailed` maybe we can catch is explicitly?